src/pages/graphql/payment-services-extension/workflows/vaulted-card.md

```diff
@@ -8,7 +8,7 @@ keywords:
 
 # Payment Services Vault workflow
 
-The following diagram shows the workflow for storing a payment method during checkout.
+The following diagram shows the workflow of a checkout authorization using a vaulted payment method.
```

Comment:
Same. The page is called "Vaulted card", yet here we generalize it to "Vaulted payment method". I propose using "Vaulted card" everywhere, which is the only vaulted payment method we support anyway.